src/XrdSecgsi/XrdSecProtocolgsi.cc

```diff
@@ -149,7 +149,7 @@ String XrdSecProtocolgsi::UsrCert  = "/.globus/usercert.pem";
 String XrdSecProtocolgsi::UsrKey   = "/.globus/userkey.pem";
 String XrdSecProtocolgsi::PxyValid = "12:00";
 int    XrdSecProtocolgsi::DepLength= 0;
-int    XrdSecProtocolgsi::DefBits  = 512;
+int    XrdSecProtocolgsi::DefBits  = 2048;
```

Comment:
This really should come from the default value define in CryptoAux which is already included in this cc file.